schema/artist/index.js

```diff
@@ -487,13 +490,15 @@ const ArtistType = new GraphQLObjectType({
         },
       },
 
-      partner_shows: Object.assign({
+      partner_shows: {
         type: new GraphQLList(PartnerShow.type),
         deprecationReason: 'Prefer to use shows attribute',
-      }, ShowField),
-      shows: Object.assign({
+        ...ShowField,
+      },
+      shows: {
         type: new GraphQLList(Show.type),
-      }, ShowField),
+        ...ShowField,
+      },
```

Comment:
from `Object.assign` to `...Thing` so that Flow can understand at compile time, instead of at runtime.